x-pack/plugins/cases/server/services/cases/index.test.ts

```diff
@@ -703,6 +774,41 @@ describe('CasesService', () => {
           res.saved_objects[0].attributes.external_service?.connector_id
         ).toMatchInlineSnapshot(`"100"`);
       });
+
+      it('properly converts the severity field to the corresponding external value the bulkPatch response', async () => {
+        unsecuredSavedObjectsClient.bulkUpdate.mockReturnValue(
+          Promise.resolve({
+            saved_objects: [
+              createCaseSavedObjectResponse({ overrides: { severity: ESCaseSeverity.LOW } }),
+              createCaseSavedObjectResponse({ overrides: { severity: ESCaseSeverity.MEDIUM } }),
+              createCaseSavedObjectResponse({ overrides: { severity: ESCaseSeverity.HIGH } }),
+              createCaseSavedObjectResponse({ overrides: { severity: ESCaseSeverity.CRITICAL } }),
+            ],
+          })
+        );
+
+        const res = await service.patchCases({
+          cases: [
+            {
+              caseId: '1',
+              updatedAttributes: createCasePostParams(getNoneCaseConnector()),
+              originalCase: {} as CaseSavedObject,
+            },
+          ],
+        });
+        expect(res.saved_objects[0].attributes.severity).toEqual(
+          SEVERITY_ESMODEL_TO_EXTERNAL[ESCaseSeverity.LOW]
+        );
+        expect(res.saved_objects[1].attributes.severity).toEqual(
+          SEVERITY_ESMODEL_TO_EXTERNAL[ESCaseSeverity.MEDIUM]
+        );
+        expect(res.saved_objects[2].attributes.severity).toEqual(
+          SEVERITY_ESMODEL_TO_EXTERNAL[ESCaseSeverity.HIGH]
+        );
+        expect(res.saved_objects[3].attributes.severity).toEqual(
+          SEVERITY_ESMODEL_TO_EXTERNAL[ESCaseSeverity.CRITICAL]
+        );
```

Comment:
nit: I think we should also check if `unsecuredSavedObjectsClient.bulkUpdate` was called with the correct severity